boilerplates/aws-lambda-hono/files/hono-entry_aws_lambda.ts

```diff
@@ -0,0 +1,34 @@
+/*
+hono-entry_aws_lambda.ts
+
+This file is the entry point for AWS Lambda
+
+Notes:
+* The file name must not have any special characters or dots except for the extension. https://docs.aws.amazon.com/lambda/latest/api/API_CreateFunction.html#API_CreateFunction_RequestSyntax
+
+*/
+
+import { Hono } from "hono";
+import { serveStatic } from "@hono/node-server/serve-static";
+import { handle } from "hono/aws-lambda";
+import type { LambdaEvent, LambdaContext } from "hono/aws-lambda";
+// @ts-ignore
+import app from "./hono-entry.js"; // file is provided by hono
```

Comment:
Bati supports importing boilerplates files like this:
```suggestion
import app from "@batijs/hono/hono-entry";
```

Reply:
`@batijs/hono` also needs to be added to the `package.json` so that it can be properly understood by IDEs and linters.